config/locales/en.yml

```diff
@@ -208,7 +214,7 @@ en:
         '3': Skip job descriptions and formal postings.
         '4': Talk directly to candidates.
         '5': Unlock private developer information.
-        '6': Hire as much or as little as you need.
+        '6': Receive email digests of new candidates.
```

Comment:
I didn't want to disrupt the 2x4 design, so I opted to just replace one of the lines. Not sure if this is preferred by you.